Gopkg.toml

```diff
@@ -0,0 +1,41 @@
+
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+required = [
+  "k8s.io/api/core/v1",
+]
```

Comment:
Trick described in #news-dev

Reply:
Why do we need tricks now?

Reply:
Otherwise master version of the `k8s.io/api` is resolved, because there is no reference in the code.

Reply:
This seems to be a general problem, what about putting this hack into operatorkit? We have to do this now everywhere where we import operatorkit.

Reply:
I tried, but it doesn't work when it's imported: https://github.com/giantswarm/operatorkit/pull/145